### DISCUSSION TITLE
Alternate indexers

### DISCUSSION THREAD

**Discussion by MrXT21** [sleeyax/stremio-easynews-addon#60] [Ideas]
Would it be possible to add the ability to search using alternate indexers rather than easynews inbuilt indexer? Alot of shows appears to be missing and using alternate programs i was able to find these programs and stream/download using easynews.

**Comment by sleeyax** [comment]
What alternate programs? Any open source ones?

**Reply by MrXT21** [reply]
None that i know of but im totally new to usenet stuff. I tried usenet panda on android and was able to add a different indexer (althub) with easynews and found everything that was missing in the default easynews search.

**Reply by sleeyax** [reply]
Ok, thanks for letting me know. This is something I could look into. Maybe the addon can adapt a similar strategy similar to the way these alternative indexers work. If you have any other pointers keep me updated.

**Comment by TaquitoSlayer** [comment]
Just bumping this on the basis of perhaps giving support to private indexers as well.
